db/version_set.cc

```diff
@@ -5510,7 +5508,8 @@ Status ReactiveVersionSet::MaybeSwitchManifest(
   Status s;
   do {
     std::string manifest_path;
-    s = GetCurrentManifestPath(&manifest_path);
+    s = GetCurrentManifestPath(&manifest_path, &manifest_file_number_,
```

Comment:
Can you run `make format` to adjust the format?

Reply:
Sure.